python/graphstorm/trainer/ep_trainer.py

```diff
@@ -31,23 +31,17 @@
 from ..utils import barrier, is_distributed, get_backend
 
 class GSgnnEdgePredictionTrainer(GSgnnTrainer):
-    """ Edge prediction trainer.
+    """ Trainer for edge prediction tasks.
 
-    This class is used to train models for edge prediction tasks,
+    This trainer is used to train models for edge prediction tasks,
     such as edge classification and edge regression.
 
-    It makes use of the functions provided by `GSgnnTrainer`
-    to define two main functions: `fit` that performs the training
-    for the model that is provided when the object is created,
-    and `eval` that evaluates a provided model against test and
-    validation data.
+    ``GSgnnEdgePredictionTrainer`` inherits from ``GSgnnTrainer`` and use the functions
+    provided by ``GSgnnTrainer`` to define two main functions: 
 
-    Parameters
-    ----------
-    model : GSgnnEdgeModel
-        The GNN model for edge prediction.
-    topk_model_to_save : int
-        The top K model to save. Default is 1.
+    * ``fit()``: performs the training for the model provided to this trainer
```

Comment:
I think we have the doc string for fit and eval. we can remove the doc string here.